dspace-server-webapp/src/main/java/org/dspace/app/rest/model/step/UploadBitstreamRest.java

```diff
@@ -16,13 +17,13 @@
 import org.dspace.app.rest.model.BitstreamFormatRest;
 import org.dspace.app.rest.model.CheckSumRest;
 import org.dspace.app.rest.model.MetadataValueRest;
-import org.dspace.app.rest.model.ResourcePolicyRest;
+import org.dspace.app.rest.model.UploadAccessConditionDTO;
 
 public class UploadBitstreamRest extends UploadStatusResponse {
```

Comment:
Could we take this opportunity to add JavaDocs to this class as well, and describe how it works with the new UploadAccessConditionDTO?  I notice the UploadAccessConditionDTO is used here & references this UploadBitstreamRest class in it's JavaDocs, but this class doesn't have it's own description